webapp/src/pages/boardPage.scss

```diff
@@ -46,3 +46,7 @@
     width: 600px;
     height: 468px;
 }
+
+.mainContentRow {
+    height: calc(100% - 40px);
```

Comment:
@harshilsharma63 should this fix be applied in `mattermost/webapp` repo? This class name is not used in `focalboard` sources.

Reply:
@kamre no it should not be in webapp. The issue is only with focalboard and not other products. The main cause of this issue is the sidebar but after spending about half an hour, I couldn't figure out what was causing the issue. This CSS  change fixes it, though not the 100% ideal solution.

Reply:
@harshilsharma63 but if the class name `mainContentRow` will be renamed in `mattermost/webapp` then this will require to update `focalboard` repo. Also just changing [this 40px](https://github.com/mattermost/mattermost-webapp/blob/0b82a79bd7ea8b52e93fa9f84bf3ac6fecedfb1f/components/global_header/global_header.tsx#L20) will also require an update here in `focalboard` repo inside this `calc` expression.

Looks like this styling is tightly coupled to `mattermost/webapp` repo. And this file `boardPage.scss` is not even inside the plugin code. So may be the issue with styling should be resolved in `mattermost/webapp` repo.

One more notice: for Playbooks that `height: 100%` for `mainContentRow` doesn't look correct, but root component in case of Playbooks has _overflow hidden_ and there is no scroll bar.

Reply:
I'm aware of this. I'll update the styling later. We're making a lot of changes to the sidebar in a separate feature branch so I don't want to spend much time here knowing the whole component is gonna change a few weeks later.